package/pyproject.toml

```diff
@@ -81,7 +81,7 @@ extra_formats = [
     "pyedr>=0.7.0",
     "pytng>=0.2.3",
     "gsd>3.0.0",
-    "rdkit>=2020.03.1",
+    "rdkit>=2021.09.1",
```

Comment:
That's ok, 2020 is already quite a long ways away. By NEP29 standards we would be dropping a December 2021 release of NumPy by January 1st 2024. So if you wanted to bump all the way up to 2022 I think that's perfectly fine (especially since it's an optional module).